src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.ClassificationVerifier.cs

```diff
@@ -0,0 +1,212 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Security.Permissions;
+using System.Windows;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Editor.ColorSchemes;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.TextManager.Interop;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.ColorSchemes
+{
+    internal partial class ColorSchemeApplier
+    {
+        private sealed class ClassificationVerifier : ForegroundThreadAffinitizedObject
+        {
+            private readonly IServiceProvider _serviceProvider;
+            private readonly ImmutableDictionary<SchemeName, ImmutableDictionary<Guid, ImmutableDictionary<string, uint>>> _colorSchemes;
+
+            private static readonly Guid TextEditorMEFItemsColorCategory = new Guid("75a05685-00a8-4ded-bae5-e7a50bfa929a");
+
+            // These classification colors (0x00BBGGRR) should match the VS\EditorColors.xml file.
+            // They are not in the scheme files because they are core classifications.
+            private const uint DarkThemePlainText = 0x00DCDCDCu;
+            private const uint DarkThemeIdentifier = DarkThemePlainText;
+            private const uint DarkThemeOperator = 0x00B4B4B4u;
+            private const uint DarkThemeKeyword = 0x00D69C56u;
+
+            private const uint LightThemePlainText = 0x00000000u;
+            private const uint LightThemeIdentifier = LightThemePlainText;
+            private const uint LightThemeOperator = LightThemePlainText;
+            private const uint LightThemeKeyword = 0x00FF0000u;
+
+            private const string PlainTextClassificationTypeName = "plain text";
```

Comment:
nit: This isn't a type name, it's a dictionary key. 
```suggestion
            private const string PlainTextIdentifier = "plain text";
```

Reply:
It is a dictionary key that happened to be the ClassificationTypeName  for "plain text"